build/configs/rtl8721csm/security_hal_test/Make.defs

```diff
@@ -21,7 +21,19 @@ include ${TOPDIR}/tools/Config.mk
 ARCH_FAMILY = $(patsubst "%",%,$(CONFIG_ARCH_FAMILY))
 include ${TOPDIR}/arch/$(CONFIG_ARCH)/src/$(ARCH_FAMILY)/Toolchain.defs
 
+ifeq ($(CONFIG_AMEBAD_TRUSTZONE),y)
+LDSCRIPT = rlx8721d_img2_ns.ld
+
+EXTRA_LIBS += ${TOPDIR}/board/rtl8721csm/src/libs/cmse_implib.a
+LINKLIBS = $(EXTRA_LIBS)
+else
 LDSCRIPT = rlx8721d_img2_is.ld
+endif
+
+ifeq ($(CONFIG_AMEBAD_TRUSTZONE),y)
```

Comment:
Here is duplicated.

Reply:
Sorry my mistake, I have remove the duplicate.